src/structures/GuildChannel.js

```diff
@@ -309,7 +312,7 @@ class GuildChannel extends Channel {
   /**
    * Set the category parent of this channel.
    * @param {GuildChannel|Snowflake} channel Parent channel
-   * @param {boolean} [options.lockPermissions] Lock the permissions to what the parent's permissions are
+   * @param {boolean} [options.lockPermissions] Lock the permissions to what the parents permissions are
```

Comment:
This is wrong

Reply:
Its wrong due to the `parents` or? Should I just revert it?

Reply:
It's grammatically wrong. You should revert it.